common/src/KokkosKernels_PrintConfiguration.hpp

```diff
@@ -18,11 +18,31 @@
 #define _KOKKOSKERNELS_PRINT_CONFIGURATION_HPP
 
 #include "KokkosKernels_config.h"
-
+#include "KokkosKernels_TplsVersion.hpp"
 #include <iostream>
 
 namespace KokkosKernels {
 namespace Impl {
+
+inline void print_cublas_version(std::ostream& os) {
+#ifdef KOKKOSKERNELS_ENABLE_TPL_CUBLAS
+  os << "  "
+     << "KOKKOSKERNELS_ENABLE_TPL_CUBLAS: " << get_cublas_version() << "\n";
+#else
+  os << "  "
+     << "KOKKOSKERNELS_ENABLE_TPL_CUBLAS: no\n";
+#endif
+}
+
+inline void print_cusparse_version(std::ostream& os) {
```

Comment:
similar comment as above

Reply:
Renamed to `print_cusparse_version_if_enabled`.